app/queries/linked_items_query.rb

```diff
@@ -0,0 +1,84 @@
+class LinkedItemsQuery
+  attr_reader :content_item
+
+  def initialize(content_item)
+    @content_item = content_item
+  end
+
+  def call
+    items = linked_items
+    items["available_translations"] = available_translations if available_translations.any?
+    if content_item.format == "working_group"
+      items["policies"] = content_items_by_id.values.select { |content_item| content_item.format == "policy" }
```

Comment:
There are two issues with this line as it stands:
* it would overwrite any policies that the item happened to link to
* it would include *any* policy returned by `content_items_by_id`, not the ones linking to the `working_group`